backend/windmill-api/src/slack_approvals.rs

```diff
@@ -998,17 +997,11 @@ async fn get_modal_blocks(
     .ok_or_else(|| error::Error::BadRequest("This workflow is no longer running and has either already timed out or been cancelled or completed.".to_string()))
     .map(|r| (r.job_kind, r.script_hash, r.raw_flow, r.parent_job, r.created_at, r.created_by, r.script_path, r.args))?;
 
-    let flow_data = match cache::job::fetch_flow(&db, job_kind, script_hash).await {
+    let flow_data = match cache::job::fetch_flow(&db, job_kind, script_hash, raw_flow).await {
```

Comment:
@alpetric Can you check this part ? I was not sure to understand the preview fetch from the parent id